inclusive-design/wecount.inclusivedesign.ca
# Galaxy fold search filter buttons (resolves #363)

## Description

Adjusted padding so search filter buttons have a better appearance on Galaxy fold screen.



## Steps to test

1. Go to deploy preview
2. Navigate to Views page 
3. Inspect webpage in Galaxy Fold screen
4. Turn on UIO syllables and open dyslexic
5. Observe search filter buttons

**Expected behavior:** 
Should be readable and all letters of a given word should be on the same line.

## Additional information
N/A


## Related issues
closes #363
